templates/specialBenefit/index.html.twig

```diff
@@ -35,6 +35,7 @@
                             {% if is_granted('ROLE_ADMIN') %}
                                 <a class="btn btn-warning mb-2 mb-md-0"
                                    href="{{ path('special_benefit_edit', {'id': specialBenefit.id}) }}">Edit</a>
+                                {{ include('specialBenefit/_delete_form.html.twig', {'specialBenefit': specialBenefit}) }}
```

Comment:
Do we really need to add the "Delete" action from the list?
In my opinion it's better the have a single delete action in the view page I don't know the opinion of the others about that?

Reply:
@ayumiesan I'd say 👍to @lbassin, can you remove the action?